app/src/androidTest/java/org/mozilla/fenix/ui/DownloadTest.kt

```diff
@@ -12,31 +13,41 @@ import org.junit.After
 import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
+import org.mozilla.fenix.customannotations.SmokeTest
 import org.mozilla.fenix.helpers.AndroidAssetDispatcher
-import org.mozilla.fenix.helpers.HomeActivityTestRule
+import org.mozilla.fenix.helpers.FeatureSettingsHelper
+import org.mozilla.fenix.helpers.HomeActivityIntentTestRule
+import org.mozilla.fenix.helpers.RetryTestRule
 import org.mozilla.fenix.helpers.TestAssetHelper
-import org.mozilla.fenix.helpers.TestAssetHelper.downloadFileName
-import org.mozilla.fenix.helpers.TestHelper
+import org.mozilla.fenix.helpers.TestHelper.deleteDownloadFromStorage
+import org.mozilla.fenix.ui.robots.browserScreen
 import org.mozilla.fenix.ui.robots.downloadRobot
 import org.mozilla.fenix.ui.robots.navigationToolbar
 import org.mozilla.fenix.ui.robots.notificationShade
 
 /**
- *  Tests for verifying basic functionality of download prompt UI
+ *  Tests for verifying basic functionality of download
  *
  *  - Initiates a download
  *  - Verifies download prompt
  *  - Verifies download notification
+ *  - Verifies various file types downloads and their appearance inside the Downloads list.
  **/
 
 class DownloadTest {
-
-    private val mDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
     private lateinit var mockWebServer: MockWebServer
+    private val mDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
+    private val featureSettingsHelper = FeatureSettingsHelper()
+    /* Remote test page managed by Mozilla Mobile QA team at https://github.com/mozilla-mobile/testapp */
+    private val testPage = "https://storage.googleapis.com/mobile_test_assets/test_app/downloads.html"
+    private var fileName: String = ""
```

Comment:
Nit: I think renaming this to `downloadFile` make it a bit clearer.

Reply:
done